sources/api/migration/migration-helpers/src/common_migrations.rs

```diff
@@ -739,3 +739,106 @@ impl Migration for ReplaceTemplateMigration {
         Ok(input)
     }
 }
+
+// =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+
+/// We use this migration when we add metadata and want to make sure they're removed before we go
+/// back to old versions that don't understand them.
+#[derive(Debug)]
+pub struct SettingsMetadata<'a> {
```

Comment:
```suggestion
pub struct SettingMetadata<'a> {
```
It's just one setting :)